bool request_scanning_for_parts
---
bool response_parts_present